Library/Homebrew/formula_installer.rb

```diff
@@ -1232,7 +1290,7 @@ def fetch_bottle_tab
                                                 .freeze
       true
     rescue DownloadError, ArgumentError
-      @fetch_bottle_tab = true
+      @fetch_bottle_tab = T.let(true, T.nilable(TrueClass))
```

Comment:
```suggestion
      @fetch_bottle_tab = T.let(true, T.nilable(T::Boolean))
```

Reply:
Hmm, this is technically correct in that the value is only ever `true` or `nil`. It's also a bit clumsy, this whole line should just be `true`.